src/medCore/gui/viewContainers/medViewContainer.cpp

```diff
@@ -168,6 +172,13 @@ medViewContainer::medViewContainer(medViewContainerSplitter *parent): QFrame(par
     connect(d->maximizedAction, SIGNAL(triggered()), this, SLOT(toggleMaximized()));
     d->maximizedAction->setEnabled(false);
 
+    d->saveSceneAction = new QAction(tr("Save scene"), d->toolBarMenu);
+    d->saveSceneAction->setToolTip(tr("Save container content as is."));
+    d->saveSceneAction->setIcon(QIcon(":icons/saveScene.png"));
```

Comment:
(or i am maybe lost. It's not that one used in pipeline. Sorry, it was a comment for the medResultsPipeline code, in music ^^')